docs(legal): move feature development policy (#6354)
zitadel/zitadel: docs/docs/legal/policies/feature-development-policy.md

---
title: Feature Development Policy
custom_edit_url: null
---

This policy clarifies how we handle requests for feature prioritization and development. This policy is applicable to situations where a user wants to prioritize certain features or development for our products and services.

## Why to do we have this policy?

Shaping of the roadmap and prioritization of features is done by ZITADEL.
You are welcome to give feedback on the [roadmap](https://zitadel.com/roadmap) and we are happy to [accept upstream pull requests](https://github.com/zitadel/zitadel/CONTRIBUTE.md) for ZITADEL.

In case you can't contribute directly to the open source version of ZITADEL, but want to accelerate development, we may develop the feature on request, under the conditions layed out in this policy.

> In are features that would go directly into our product, any of the publicly available ZITADEL [repositories](https://github.com/zitadel/), our cloud service, or optional components to our products that might be available only under a commercial license.

Reviewer: Something looks strange with this sentence
"In are features that..."? But not sure what is meant with it

Author: ```suggestion
Features that will be part of the publicly available ZITADEL [repositories](https://github.com/zitadel/), ZITADEL Cloud, or optional components to our products that might be available only under a commercial license.
```
